### PR TITLE
Converted_Images_to_WebP_for_Better_Optimization

### PR DESCRIPTION
**Pull request by smog-root**
**Fixes** #309 

- Converted all relevant images from formats like PNG and JPEG to WebP for better optimization and performance.
- WebP offers better compression without significant quality loss, which helps in reducing file sizes and improves page load speed.

**Changes:**

- Replaced existing image formats (JPEG, PNG) with WebP versions.
- Updated image references in HTML files to reflect the new WebP file paths.
- Verified that images render correctly in browsers that support WebP.

**Testing:**

- Verified on multiple browsers (Chrome, Firefox, etc.) to ensure proper image rendering.
- Ensured fallback mechanisms are in place for browsers that do not support WebP (if needed).

_**Note**_:  Add gssoc-ext with level3, hacktoberfest-accepted labels! I think this PR deserves level3 label. Please add it i have worked hard to do this! 

@Avdhesh-Varshney

### CODE REVIEW

**Comment by smog-root** [comment]
Add labels and assign me! then merge !

File: public/arthashastra.webp
Comment (Avdhesh-Varshney) [inline public/arthashastra.webp:1]: .

File: public/favicon.ico
Comment (Avdhesh-Varshney) [inline public/favicon.ico:1]: .

File: public/image.webp
Comment (Avdhesh-Varshney) [inline public/image.webp:1]: .

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
Don't covert logo icon from png to webp. Just convert only high resolution images.
- Review your deployment on your own

**Comment by smog-root** [comment]
Okay I will convert back logo form webp back to png! @Avdhesh-Varshney

**Comment by smog-root** [comment]
@Avdhesh-Varshney merge it!!!

**Comment by smog-root** [comment]
@Avdhesh-Varshney ??

File: public/android-chrome-192x192.png
Comment (Avdhesh-Varshney) [inline public/android-chrome-192x192.png:1]: .

File: public/android-chrome-512x512.png
Comment (Avdhesh-Varshney) [inline public/android-chrome-512x512.png:1]: .

File: public/apple-touch-icon.png
Comment (Avdhesh-Varshney) [inline public/apple-touch-icon.png:1]: .

File: public/favicon-16x16.png
Comment (Avdhesh-Varshney) [inline public/favicon-16x16.png:1]: .

File: public/favicon-32x32.png
Comment (Avdhesh-Varshney) [inline public/favicon-32x32.png:1]: .

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
These all are icons not the images

**Comment by smog-root** [comment]
@Avdhesh-Varshney I will open another PR with changes!!!

**Pull request closed without merging**